### PR TITLE
Feature/tls improvements 

### PR DESCRIPTION
**Pull request by lenapivotal**
Collection of changes from the dev EPIC around TLS, after merging a PR from Andrew. It also includes release notes for this topic. 

Html app:
https://gp-docs-lena-mysql120-tls.sc2-04-pcf1-apps.oc.vmware.com/tanzu-mysql-kubernetes/1-1/configure-tls.html

### CODE REVIEW

File: configure-tls.html.md.erb [hunk 1]
```diff
@@ -3,24 +3,161 @@ title: Configuring TLS for MySQL Instances
...
+- By default, the Tanzu MySQL Operator configures a self-signed `ClusterIssuer` for issuing MySQL instance TLS certificates. See [Using the Default provided TLS](#using-default-tls). 
+
+- Administrators can also customize the Certificate Authority (CA) to comply with an organization's CA. See [Configuring a Custom TLS Issuer](#configuring-custom-tls-issuer).
+
+- MySQL server administrators can override the default or the custom CA, by using a per instance TLS key pair. See [Explicitly Configure a MySQL instance for TLS](#explicitly-configure-instance-for-TLS).
+
+
+## <a id="using-default-tls"></a>Using the Default Provided TLS
+
+From Tanzu MySQL 1.2.0, the MySQL Operator deploys the instances with TLS from the self-signed `ClusterIssuer` configured during Operator install.
```
Comment (ricardo-vm) [inline configure-tls.html.md.erb:23]: Should this version be a variable, or is it meant as "From Tanzu MySQL 1.2.0 onwards..."?

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -43,7 +66,7 @@ The <%= vars.product_short %> releases support the components listed below:
 * Release 1.1.0 supports MySQL Server 8.0.25. For details on MySQL Server 8.0.25 see [MySQL Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html) and [Percona Server Release Notes](https://www.percona.com/doc/percona-server/LATEST/release-notes/Percona-Server-8.0.25-15.html).
 * Release 1.1.0 supports Percona XtraBackup 8.0.25. For details on Percona XtraBackup 8.0.25 see [Percona XtraBackup Release Notes](https://www.percona.com/doc/percona-xtrabackup/LATEST/release-notes/8.0/8.0.25-17.0.html).
 * Tanzu MySQL instances are now created with a version number. Users may specify the `spec.version` field in the instance definition to specify which Tanzu MySQL version to deploy. Users can now list instances by version number. For more details, see [List Instances By Version](upgrade-instance.html).
-* When installing <%= vars.product_full %>, users can specify a default version for new Tanzu MySQL instances. For more details, see [Installing the Operator](./install-operator.html).
+* When installing Tanzu MySQL, users can specify a default version for new Tanzu MySQL instances. For more details, see [Installing the Operator](./install-operator.html).
```
Comment (ricardo-vm) [inline release-notes.html.md.erb:69]: Should this be the vars.product_short?

File: install-operator.html.md.erb [hunk 3]
```diff
@@ -416,11 +425,15 @@ resources: {}
...
 1. Edit the values that you want to change.
 1. Delete the sections of the file that you do not change.
-1.  Save the `operator-values-overrides.yaml` file in a location of your choice or the same directory as the `values.yaml` file.
+1. Save the `operator-values-overrides.yaml` file in a location of your choice or the same directory as the `values.yaml` file.
 
 #### <a id="create-release"></a>Create the MySQL Operator 
-    
-1.  Use Helm to install the MySQL Operator in your Kubernetes cluster.
+
+**IMPORTANT**: Helm 3.7 is currently not supported.
```
Comment (ricardo-vm) [inline install-operator.html.md.erb:432]: Consider changing to 3.7.x.

File: install-operator.html.md.erb [hunk 4]
```diff
@@ -222,14 +227,18 @@ replicaset.apps/mysql-for-kubernetes-controller-manager-84d76dfb77   1         1
...
-## <a id="install-via-tar"></a>Install Tanzu Operator via a Downloaded Archive File
+## <a id="install-via-tar"></a>Install the Operator via a Downloaded File
         
 Choose this method if:
 
 - The installation destination (for example an air-gapped network) cannot access the [VMware Tanzu Network](https://network.pivotal.io).
 - Or you wish to load the Operator and instance images to a private Docker registry.
 
+**IMPORTANT**: Helm 3.7 is currently not supported.
+
```
Comment (ricardo-vm) [inline install-operator.html.md.erb:241]: Consider changing to 3.7.x

File: install-operator.html.md.erb [hunk 5]
```diff
@@ -24,7 +22,8 @@ Before you deploy the Tanzu MySQL Operator, you need:
 
 - The [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) command-line tool, configured and authenticated to communicate with your Kubernetes cluster.
 
-- The Helm v3 command-line tool installed. For more information, see [Installing Helm](https://helm.sh/docs/intro/install/) from the Helm documentation.
+- The Helm v3 command-line tool installed. For more information, see [Installing Helm](https://helm.sh/docs/intro/install/) from the Helm documentation. <br>
+    **IMPORTANT**: Helm 3.7 is currently not supported.
```
Comment (ricardo-vm) [inline install-operator.html.md.erb:26]: Consider changing to "3.7.x" (sorry for the repeated comments, I started bottom-up)

File: configure-tls.html.md.erb (hunk 1 above)
Comment (lenapivotal) [reply configure-tls.html.md.erb:23]: It is meant as 1.2.0 and later... I'll make the change

File: release-notes.html.md.erb (hunk 2 above)
Comment (lenapivotal) [reply release-notes.html.md.erb:69]: I've had some issues with variables and publishing (they don't appear properly) so I'm starting to avoid them :)

**Pull request merged**